app/controllers/app_groups_controller.rb

```diff
@@ -4,8 +4,9 @@ class AppGroupsController < ApplicationController
 
   def index
     @app_groups = policy_scope(AppGroup).order(:name)
-    @allow_create_app_group = policy(Group).index?
-    @allow_set_status = policy(Infrastructure).toggle_status?
+    @allow_create_app_group = policy(AppGroup).new?
+    # @allow_set_status = policy(Infrastructure).toggle_status?
+    @is_barito_superadmin = policy(Group).index?
```

Comment:
change the name of the variable, `is_barito_superadmin` is not reflecting on what it does

Reply:
done